app/services/user.py

```diff
@@ -0,0 +1,19 @@
+from advanced_alchemy.extensions.litestar import repository, service
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app import models as m
```

Comment:
Very small nit: i'm personnally not very fond of one-letter variable names. I'd rather we just used `models` everywhere, but really, it's up to you.

If you decide on changing it, please also change it in the other places ;)